### ISSUE TITLE
macOS text replacement shortcuts don't work

### ISSUE THREAD

**Issue by spurll** [Automattic/simplenote-electron#630]
macOS provides built-in, customizable "replace-as-you-type" functionality. For example, you can easily set things up such that a text shortcut is replaced with full text or symbols. (Just go to System -> Keyboard -> Text.)

These text replacements are supported by the vast majority of applications in the Mac ecosystem, and they are very helpful. I do a lot of stats (and I'm also currently a chemistry student), and it's helpful to be able to quickly insert a Greek sigma or nu (for example). But because Simplenote for Mac doesn't support this feature, I need to open up another text editor (or just Spotlight) and copy/paste from there.

FWIW, text replacements work just fine in the iOS version of Simplenote. (With the keyboard handling being so different, though, I don't know if this information is helpful.)

This issue is similar to #334.

**Comment by adlk** [comment]
_Note to myself:_ It is technically possible with this module https://github.com/CharlieHess/electron-text-substitutions

**Comment by mirka** [comment]
Thanks for the report! I looked into this, and it is a limitation with Chromium. (Electron apps are based on it)
https://apple.stackexchange.com/questions/179187/turn-on-text-replacement-chrome

I could confirm that while third-party utilities like TextExpander and Typinator work, built-in macOS text replacements do not work in Chrome.

I found [a module](https://github.com/CharlieHess/electron-text-substitutions) that works around this, but it [doesn't seem stable](https://github.com/CharlieHess/electron-text-substitutions/issues) and it currently doesn't work with Electron 4.0, so it's probably not a good solution here.

Closing as a `wontfix` for now. Let's hope Chromium supports it in the near future 🤞
